AmplifyTools/AmplifyXcode/Sources/AmplifyXcode/main.swift

```diff
@@ -0,0 +1,10 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+AmplifyXcode.main()
```

Comment:
Silly question, but where is this declared?

Reply:
the `main` method comes from [Apple's ArgumentParser lib](https://github.com/apple/swift-argument-parser) :)